app/api/utility.py

```diff
@@ -40,6 +41,7 @@ def create_query(
     sex: str = None,
     diagnosis: str = None,
     is_control: bool = None,
+    min_num_sessions: int = None,
```

Comment:
I think the correct way here would be to use 

```python
from typing import Optional
min_num_sessions: Optional[int] = None
```

because we are using type hinting and `None` as a default for an `int`. See here: https://docs.python.org/3/library/typing.html#typing.Optional

This is true of the other stuff as well, and in other places, so maybe that's something for another PR.

Reply:
Yea I think it's better to correct this in another issue.